### ISSUE TITLE
"truststore" secret type breaks secretmanager.EnsureSecrets

### ISSUE THREAD

**Issue by jrcast** [ForgeRock/secret-agent#48]
Seems there's a problem when processing `truststore` with `caCopy`
using a quick debug print:
```
// add secret version
valueString := string(node.Value)
fmt.Printf("The length of %q is %d\n", secretID, len(valueString)) //DEBUG PRINT
```
printed
```
The length of "default-amster-authorized-keys" is 381
The length of "default-truststore-cacerts" is 2306
The length of "default-truststore-cacerts-ca-cert" is 0 <<< ca-cert seems to be empty at the moment it is EnsureSecrets is called.
```
You can't create secrets with length=0 

removing `truststore` from the `sac` allows the secretmanager to work again.

**Comment by jrcast** [comment]
This issue should be fixed as part of #111

**Comment by jrcast** [comment]
See #113
This feature is no longer needed
